recipes-bsp/u-boot/u-boot_%.bbappend

```diff
@@ -7,6 +8,20 @@ SRC_URI:append:freedom-u540_sota = " file://uEnv.txt"
 
 DEPENDS:append:freedom-u540 = " u-boot-tools-native"
 
+SRC_URI:append:ae350-ax45mp = " \
+            file://0001-Fix-SD-detect-failure.patch \
+            file://0002-Prevent-fw_dynamic-relocation.patch \
+            file://0003-riscv-ae350-Fix-XIP-config-boot-failure.patch \
+            file://0004-spl_opensbi.c-enable-printing-OpenSBI-boot-log.patch \
+            file://0005-mmc-ftsdc010_mci-Support-DTS-of-ftsdc010-driver-for-.patch \
+            file://u-boot.cfg \
+            file://tftp-mmc-boot.txt \
+            file://uEnv-ae350.txt \
+            "
+
+DEPENDS:append = " u-boot-tools-native"
```

Comment:
Do you actually need this?

Reply:
Yes, u-boot will fail to configure (like the error shown below) without `u-boot-tools-native` since I need `mkimage` to generate boot.scr.uimg. Can I share the variable with freedom-u540?

```
|     /work/poky/meta/recipes-bsp/u-boot/u-boot/                                                                       
|     /work/poky/meta/recipes-bsp/u-boot/files/                                                                        
| DEBUG: Executing shell function do_configure                                                                         
| /build/tmp/work/ae350_ax45mp-poky-linux/u-boot/1_2022.07-r0/temp/run.do_configure.52010: 150: mkimage: not found     
| WARNING: exit code 127 from a shell command.                                                                         
ERROR: Task (/work/poky/meta/recipes-bsp/u-boot/u-boot_2022.07.bb:do_configure) failed with exit code '1'
NOTE: Tasks Summary: Attempted 2689 tasks of which 2651 didn't need to be rerun and 1 failed.
                                                                                                                       
Summary: 1 task failed:                                                                                                
  /work/poky/meta/recipes-bsp/u-boot/u-boot_2022.07.bb:do_configure
```